### PR TITLE
Foreman 1.24.1

### PR DESCRIPTION
**Pull request by pcreech**

### CODE REVIEW

File: packages/foreman/foreman-installer/foreman-installer.spec [hunk 1]
```diff
@@ -139,6 +139,9 @@ done
 %{_sbindir}/foreman-proxy-certs-generate
 
 %changelog
+* Thu Jan 09 2020 b'Patrick Creech <pcreech@redhat.com>' - b'1:1.24.1-1'
```
Comment (tbrisker) [inline packages/foreman/foreman-installer/foreman-installer.spec]: looks like you're hitting the same issue i noticed when bumping develop to 2.0 - these `b'` tags seem to indicate some python string encoding problem in obal and should be dropped like so:
```suggestion
* Thu Jan 09 2020 Patrick Creech <pcreech@redhat.com> - 1:1.24.1-1
```

File: packages/foreman/foreman-release/foreman-release.spec [hunk 2]
```diff
@@ -114,6 +114,9 @@ install -Dpm0644 %{SOURCE4} %{buildroot}%{_sysconfdir}/pki/rpm-gpg/RPM-GPG-KEY-f
 %{_sysconfdir}/pki/rpm-gpg/RPM-GPG-KEY-foreman-rails
 
 %changelog
+* Thu Jan 09 2020 b'Patrick Creech <pcreech@redhat.com>' - b'1.24.1-1'
```
Comment (tbrisker) [inline packages/foreman/foreman-release/foreman-release.spec]: missed this one

**Comment by pcreech** [comment]
el5 was rebuilding srpms.  I disabled the rebuildSRPM step and am rerunning

**Comment by pcreech** [comment]
[test]

**Comment by pcreech** [comment]
[test rpm]

**Pull request merged**